src/main/resources/userEmailTemplate-feedbackSessionClosed.html

```diff
@@ -3,6 +3,7 @@
 <p>
     The following session is now closed. You can still view your submission by going to the link sent earlier,
     but you will not be able to edit existing responses or submit new responses.
+    <strong>No action is required if you have already submitted.</strong>
```

Comment:
Perhaps put this in line 13 instead? It goes together with instructions on how to request extensions.